client/components/button-fill.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <component
+    class="button-fill"
+    :href="href"
+    :is="tag"
+    :to="to"
+    @click="onClick"
+  >
+    {{ label }}
+  </component>
+</template>
+
+<script>
+export default {
+  name: 'button-fill',
+  props: {
+    href: {
+      type: String,
+    },
+    label: {
+      type: String,
+    },
+    tag: {
+      type: String,
+      default: 'button',
+    },
+    to: {
+      type: Object,
+    },
+  },
+  methods: {
+    onClick(...args) {
+      this.$emit('click', ...args);
+    },
+  },
+};
+</script>
+
+<style lang="stylus">
+.button-fill {
+  cursor: pointer;
```

Comment:
If HTML element is semantic, then you shouldn't need `cursor: pointer` - right?

Reply:
If it uses router-link it doesn't show cursor for some reason.

Reply:
Weird, that's a miss on router-link then. Is there an open issue for it?